### DISCUSSION TITLE
disable bluetooth at sturtup

### DISCUSSION THREAD

**Discussion by akrram09** [blueman-project/blueman#2587] [Q&A]
blueman:2.4.3-2
BlueZ:5.79-1
Distribution:Arch
Desktop environment:SwayWM

i want to disable turn on bluetooth at startup tried [this on ArchWiki](https://wiki.archlinux.org/title/Blueman#Disable_auto_power-on)
but i got
`No such schema “org.blueman.plugins.powermanager”`

Then, as a last resort, I tried [this](https://wiki.archlinux.org/title/Bluetooth#Default_adapter_power_state). After reboot, it still didn't disable it.

**Comment by infirit** [comment]
blueman removed the auto powering of the adapter a while ago. As for why BlueZ isn't setting the correct power state you'll have to double check your changes and if you can't get it to work ask on the arch forums.
